src/types/license.types.ts

```diff
@@ -29,7 +59,7 @@ export interface File {
   administrativeContract?: boolean;
   isPublished?: boolean | null;
   embargoDate?: Date | null;
-  license?: License | null;
+  license?: string;
```

Comment:
hvorfor brukes ikke License-typen her lengre?

Reply:
Typen ble lagd på slutten av september, og den ble ikke brukt annen plass enn her, og License blir ikke satt for filer før denne PR, så usikker på hva som har skjedd der